callback_plugins/junit_callback.py

```diff
@@ -0,0 +1,354 @@
+# (c) 2016 Matt Clay <matt@mystile.com>
+# (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# Forked and modified by Sagi Dayan <sdayan@redhat.com>
```

Comment:
Could you specify which commit/file you forked from? So we can easily re-apply those changes to newer version and if someone is interested they could also compare to learn what exactly you changed

Reply:
A nice overview of what you changed and why would be nice too

Reply:
Good point. Adding info